rotkehlchen/data_import/utils.py

```diff
@@ -75,9 +97,45 @@ def flush_all(self, write_cursor: DBCursor) -> None:
         self._asset_movements = []
         self._history_events = []
 
+    def append_msg(self, row_index: int, msg: str, warning: bool) -> None:
+        """Append message to queue to be sent to frontend"""
+        if self.max_msgs:
+            return
+
+        # Stop collecting msgs if more than MAX_ERROR_PERCENT have errors.
+        # Note that total_entries actually only contains the number of rows processed do far.
+        # So if the percentage of errors is too high in the first MIN_ENTRIES rows
+        # error collecting will stop even if the rest of the file has few errors.
+        if (
+            self.total_entries >= MIN_ENTRIES and
+            len(self.import_msgs) >= self.total_entries * MAX_ERROR_PERCENT
+        ):
+            self.max_msgs = True
+
+        self.import_msgs.append({
+            'row': row_index,
+            'msg': msg,
+            'warning': warning,
+        })
+
+    def log_debug(self, row_index: int, csv_row: dict, msg: str) -> None:
+        log.debug(f'Skipping row {row_index} of {self.name} CSV file. {msg}')
+        log.debug(csv_row)
+        self.append_msg(row_index, msg, False)
```

Comment:
similarly in other places where we call them, because reader doesn't know what False is here
```suggestion
        self.append_msg(row_index=row_index, msg=msg, warning=False)
```